pubnub-hyper/src/transport/hyper/presence.rs

```diff
@@ -134,8 +134,8 @@ impl TransportService<request::SetState> for Hyper {
             state,
         } = request;
 
-        let channels = EncodedChannelsList::from(channels);
-        let channel_groups = EncodedChannelsList::from(channel_groups);
+        let channels = UrlEncodedList::from(channels);
+        let channel_groups = UrlEncodedList::from(channel_groups);
```

Comment:
Can these reuse `pubsub::process_subscribe_to()`?

Reply:
It is unclear if the logic is the same. I decided to hold on the decision to change these calls until we have more calls - and then either generalize the `SubscribeTo`, or implement more similar objects.